server/interpreter.js

```diff
@@ -3386,4 +3386,7 @@ Interpreter.prototype['stepWithStatement'] = function() {
 Interpreter.prototype['stepWhileStatement'] =
     Interpreter.prototype['stepDoWhileStatement'];
 
-module.exports = Interpreter;
+if (typeof module !== 'undefined') { // Node.js
```

Comment:
Wontfix.  See https://groups.google.com/a/google.com/forum/#!search/javascript$20equality%7Csort:date/javascript-style/wSOzSX6BCKw/6gVzHM2-DgAJ for the most recent discussion I can find on this, but in summary:

* `==` was previously (ca. 2007) mandated by the styleguide but has not been for some time (i.e., in at least the current and most recent previous editions).
* Most of those arguing in favour of `==` by default are:
   * now only making that argument in only respect to large existing google3 code bases where I agree that it does offer some readability benefits (`interpreter.js` is large but not that large yet...), and
   * assuming the code is going  to be compiled (ours will not be, AFAIK) and so will be type-checked.
* Even many newer google3 JS projects now mandate `===`.
* Outside of Google`===` is effectively the industry standard.

We should be using it in-core for pedagogical reasons, and in the server implementation because we are an open-source project and should be facilitating external contributions.